modules/dists/ReplicatedDist.chpl

```diff
@@ -424,6 +424,16 @@ class ReplicatedArr : BaseArr {
         halt("locale ", here.id, " has no local replicand");
     return localArrs[here.id];
   }
+
+  /* Access another locale's local array representation by locale id */
+  proc replicand(id: here.id.type) ref {
```

Comment:
Good point. I'm happy with `getReplicand(loc: locale) { }`

>Completely unrelated to this PR, I"m noting that ref doesn't get highlighted above. :'(

Might be worth tracking in a new "Improve TmBundle Support" issue (like we have for pygments in #5900)